Dockerfile

```diff
@@ -74,6 +72,11 @@ RUN apt-get update && \
     npm update && \
     npm install --silent -g bower less clean-css uglify-js requirejs
 
+# add invenio user
+RUN useradd --home-dir /home/invenio --create-home --shell /bin/bash --uid 1000 invenio && \
+    echo "alias ll='ls -alF'" >>  /home/invenio/.bashrc && \
+    echo "invenio ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers
```

Comment:
Giving `invenio` superuser rights is dangerous.  It may be good for development, but it's bad for production.  Note that many services build on top of Invenio docker image, see `FROM invenio:2.0` and friends in our overlays. (e.g.  `invenio-demosite`, `opendata.cern.ch`, `analysis-preservation.cern.ch`)